scss/mixin.scss

```diff
@@ -43,3 +43,30 @@
     }
   }
 }
+
+@mixin homepage-search-dropdown( $border-color, $home-dropdown ) {
+  @include scaleText(16px ,24px ,20px ,28px);
```

Comment:
the comma-spacing format is weird here. it should be `, ` instead of ` ,` :)

Reply:
Lol I noticed that after the fact, thanks haha